docs/azure/DeviceCredentials.rst

```diff
@@ -1,77 +1,90 @@
 .. _azure-device-credentials:
 
-Device Credentials
+Device credentials
 ##################
 
-On Azure devices connect to IoT Hub using `TLS version 1.2 <https://docs.microsoft.com/en-us/azure/iot-fundamentals/iot-security-deployment>`_ and Elliptic Curve Cryptography (ECC) based certificates.
+On Azure devices, connect to IoT Hub using `TLS version 1.2 <https://docs.microsoft.com/en-us/azure/iot-fundamentals/iot-security-deployment>`_ and Elliptic Curve Cryptography (ECC) based certificates.
 
 Azure supports offline generation of device certificates through `Enrollment Groups <https://docs.microsoft.com/bs-latn-ba/azure/iot-dps/quick-enroll-device-x509-node>`_.
 
 Device certificates are signed using an intermediate CA.
 This allows to keep the CA root certificate secure and create an intermediate CA certificate to be used during device provisioning.
-This way the CA root certificate private key will never be transmitted to insecure locations (e.g. a third-party factory).
+In this case, CA root certificate private key will never be transmitted to insecure locations (for example, a third-party factory).
 
-If the intermediate CA certificate is compromised, it can be deactivated so new devices with credentials signed by the compromised intermediate CA certificate will no longer be provisioned.
+If the intermediate CA certificate is compromised, it can be deactivated.
+New devices with credentials signed by the compromised intermediate CA certificate will no longer be provisioned.
 
-Devices that previously connected to the IoT Hub, will keep working.
+Devices that previously connected to the IoT Hub, will continue to work.
 
-Device Provisioning Service (DPS)
-*********************************
+For creating device credentials, you must generate the following certificates:
```

Comment:
I find this a little confusing, because the next section is about the DPS.

![image](https://user-images.githubusercontent.com/188915/106742262-1954ba80-661d-11eb-9a38-42d25ce63db7.png)

So how about you move that list of steps before the individual sections?

## Creating device credentials

For creating device credentials, you must generate the following certificates:

- CA root certificate
- CA intermediate certificate
- Device certificate

### CA root certificate

... 

### CA intermediate certificate

... 

### Device certificate

...